larq/metrics.py

```diff
@@ -36,12 +36,6 @@ def __init__(self, values_dtype="int8", name="flip_ratio", dtype=None):
         self.built = False
         self.values_dtype = tf.as_dtype(values_dtype)
 
-    def __call__(self, inputs, **kwargs):
-        if not self.built:
-            with tf.name_scope(self.name), tf.init_scope():
-                self.build(inputs.shape)
-        return super().__call__(inputs, **kwargs)
```

Comment:
For reference `super().__call__` will implicitely call `update_state` so we don't need to overwrite it here.